### DISCUSSION TITLE
Feature merge window

### DISCUSSION THREAD

**Discussion by sreimers** [baresip/baresip#2237] [General]
Should we introduce some kind of feature merge window? At the moment we have a four week release cycle. Should we only merge new features the first two weeks and after that only accept bug fixes? I think this would help to make releases more stable.

Example:
- Oct 18 2022 - Feature freeze
- Nov 1 2022 - Stable release

**Comment by juha-h** [comment]
Sebastian Reimers writes:

> Should we introduce some kind of feature merge window? At the moment
> we have a four week release cycle. Should we only merge new features
> the first two weeks and after that only accept bug fixes? I think this
> would help to make releases more stable.

New feature freeze before releases makes sense, but I wonder if the
current four week release cycle is too short.

**Reply by alfredh** [reply]
I agree about the feature freeze.

Should we increase the release cycle to 5 weeks ?

**Reply by sreimers** [reply]
5 weeks is also fine for me, should we prefer a weekday, like wednesday?

**Comment by alfredh** [comment]
hi @sreimers 

just to double check, is the next release planned for 1. Nov 2022 ?

**Reply by sreimers** [reply]
Yes I think so, no big changes in the last 14 days. After this we can try a increased release cycle if wanted/needed.

**Comment by alfredh** [comment]
I think we can deprecate or remove Makefile from baresip after this release.

Is there anyone who is objecting to that ?
